tests/rpc_test.go

```diff
@@ -52,6 +57,22 @@ type Response struct {
 	Result json.RawMessage `json:"result,omitempty"`
 }
 
+func TestMain(m *testing.M) {
+	if ETHERMINT_INTEGRATION_TEST_MODE != "stable" {
```

Comment:
does "live" mode do anything right now?

Reply:
hmm....no

Reply:
okay can we remove it then?  it's cleaner to add vars when you need them

Reply:
the objective with this check is to run only tests that are executed with the env var is set. It allow us to profile the tests and avoid missing it with normal unit tests.